frontend/.lintstagedrc.js

```diff
@@ -1,5 +1,14 @@
 module.exports = {
```

Comment:
Should we use shared lint-staged config or should we have config per package?

If it is shared probably all of the lint/format packages should be shared as well